app/models.py

```diff
@@ -1632,9 +1631,9 @@ class NotificationHistory(db.Model, HistoryModel):
     reference = db.Column(db.String, nullable=True, index=True)
     client_reference = db.Column(db.String, nullable=True)
 
-    international = db.Column(db.Boolean, nullable=False, default=False)
+    international = db.Column(db.Boolean, nullable=True, default=False)
```

Comment:
Is there protection in place to capture null (an empty string) where this code is called/used?

Reply:
How does changing this to `nullable=True` address the issue raised with the unexpected output?

Reply:
@cris-oddball 
```
op.alter_column('notification_history', 'international',
               existing_type=sa.BOOLEAN(),
               nullable=False)
```